kotsadm/pkg/handlers/config.go

```diff
@@ -27,7 +28,9 @@ import (
 	versiontypes "github.com/replicatedhq/kots/kotsadm/pkg/version/types"
 	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
 	"github.com/replicatedhq/kots/kotskinds/multitype"
+	config2 "github.com/replicatedhq/kots/pkg/config"
```

Comment:
how about naming this `kotsconfig` and the other one `kotsadmconfig`? i feel like it would be clearer

Reply:
sure
this is what happens when I rely on IDE autoimports